### DISCUSSION TITLE
Spark job getting killed on execution

### DISCUSSION THREAD

**Discussion by ArijitSinghEDA** [hussein-awala/spark-on-k8s#86] [Q&A]
In regards to #85 

The code gets executed, but the moment it starts running, the spark jobs start getting killed automatically. This behavior is being observed when trying to execute file which is on a mounted path

Is it a known error or there is some configuration still needed to be done?

**Comment by hussein-awala** [comment]
As I explained [here](https://github.com/hussein-awala/spark-on-k8s/discussions/87#discussioncomment-10136207), there is a problem in the provided parameters, but if you already fixed it and you still have the same issue, could you share the driver log when it is killed?
